examples/logging-demo/Makefile

```diff
@@ -1,34 +1,26 @@
 # Makefile for launching syntheitc logging sources (any platform)
 # and for reporting the forwarding rules for the
 # Elasticsearch and Kibana pods for the GCE platform.
+# For examples of how to observe the ingested logs please
+# see thea appropriate getting started guide e.g.
+# Google Cloud Logging: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/logging.md
+# GCE with Elasticsearch and Kibana: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/logging-elasticsearch.md
```

Comment:
Do you really need to qualify this as being for GCE? It's very generic to different cloud providers other than the output of the cluster-up command.